portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/antisamy/AntiSamy.testcase

```diff
@@ -128,6 +120,8 @@ definition {
 					</script>
 				''';
 
+			Navigator.openSpecificURL(url = "http://localhost:8080/web-content-display-page");
```

Comment:
Hi @Kvale1733 ,

These tests are also executed on cloud envs such as "http://cloud-10-0-13-63.lax.liferay.com:8080" so we cannot hardcode the URL to localhost, as it will cause test failure.

Instead, we could get/define the portal URL, and use the ${portalURL} parameter on such URLs.
Also, it could happen we need to send/define the portal URL to the macro calls also.

Same test case [result](https://testray.liferay.com/reports/production/logs/2021-11/test-1-6/test-portal-testsuite-upstream(master)/833/functional-tomcat90-mysql57-jdk8/1/2/LocalFile.AntiSamy_ScriptInWebContentDisplayPortletShouldBeSanitized/summary.html.gz) on recent daily run